R/srr-stats-standards.R

```diff
@@ -0,0 +1,132 @@
+#' srr_stats
+#'
+#' All of the following standards initially have `@srrstatsTODO` tags.
+#' These may be moved at any time to any other locations in your code.
+#' Once addressed, please modify the tag from `@srrstatsTODO` to `@srrstats`,
+#' or `@srrstatsNA`, ensuring that references to every one of the following
+#' standards remain somewhere within your code.
+#' (These comments may be deleted at any time.)
+#'
+#' @srrstatsVerbose TRUE
+#'
+#' @srrstatsTODO {G1.2} *Statistical Software should include a* Life Cycle Statement *describing current and anticipated future states of development.*
+#' @srrstats {G1.4} The package uses [`roxygen2`](https://roxygen2.r-lib.org/) to document all functions.
+#' @srrstatsTODO {G1.4a} *All internal (non-exported) functions should also be documented in standard [`roxygen2`](https://roxygen2.r-lib.org/) format, along with a final `@noRd` tag to suppress automatic generation of `.Rd` files.*
```

Comment:
I am not sure if this means adding full documentation to all internal functions or only making sure that both the `@noRd` and `@srrstats {G1.4a}` tags are added.

Reply:
Good point, I think the former..

Reply:
@fnattino, I implemented this for all internal functions. It felt redundant to add descriptions so I only added the title if there was no other documentation besides the `@noRd` tag.